MissingLetterTest.kt

```diff
@@ -0,0 +1,29 @@
+import com.sun.tools.javac.Main
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+internal class MainKtTest {

```

Comment:
Same comment as for problem 1, one interesting unit test is what happens if there is no missing letter?

Reply:
Testing it now, for [a,b,c,d] it would return f